server-provider/src/main/java/org/sonatype/tests/http/server/jetty/impl/MonitorableProxyServlet.java

```diff
@@ -70,7 +70,7 @@ public void service(ServletRequest req, ServletResponse res)
       String proxyAuthorization = request.getHeader("Proxy-Authorization");
       if (proxyAuthorization != null && proxyAuthorization.startsWith("Basic ")) {
         String proxyAuth = proxyAuthorization.substring(6);
-        String authorization = B64Code.decode(proxyAuth, StringUtil.__ISO_8859_1);
+        String authorization = new String(Base64.getDecoder().decode(proxyAuth.getBytes()), StringUtil.__ISO_8859_1);
```

Comment:
Trivial: getBytes() should be explicit about the codeset (UTF-8) in case of platform settings.

Reply:
> Trivial: getBytes() should be explicit about the codeset (UTF-8) in case of platform settings.

Looking at it again, I don't think this should even use `getBytes()` I think it should probably simply pass the string.